src/components/events/EventForm.jsx

```diff
@@ -76,7 +89,7 @@ class EventForm extends React.Component {
         <div style={{ margin: '25px' }}>
           <DialogInput fullWidth name="eventTitle" label="Event Title" onChange={this.handleChange} value={eventTitle} />
           <div style={{ display: 'flex' }}>
-            <DialogInput style={{ marginRight: '5px' }} fullWidth name="eventDate" label="Event Date" onChange={this.handleChange} placeholder="DD/MM/YYYY" value={eventDate} />
+            <TextField  fullWidth name='eventDate' label='Event Date' onChange={this.handleChange} style={{ marginRight: '5px' }} type="date" value={eventDate ? this.convertDateToISOFormat(eventDate) : ''} variant="filled" />
```

Comment:
Why do we need to convert this from a DialogInput to a TextField? Is a DialogInput not our implementation of the MaterialUI Textfield?